pages/securing.rst

```diff
@@ -11,6 +11,27 @@ Never run this in an insecure network.
 
 When using Amazon Web Services and our pre-configured AMI, never open all ports in the security group. Do not expose the server to the internet. Access Graylog only from within your VPC. Enable encryption for the communication.
 
+Default ports
+=============
+
+All parts of one Graylog installation will communicate over network sockets. Depending on your setup and number of nodes this might be exposed or can be bound to localhost. The :ref:`SELinux <selinux>` configuration is already covered in our step-by-step guide for CentOS Linux. 
+
+.. list-table:: Default network communication ports
+    :header-rows: 1
+
+    * - Component
+      - Port
+    * - Graylog (web interface / API)
+      - 9000 (tcp)
+    * - Elasticsearch (binary protocol)
+      - 9300 (tcp)
```

Comment:
Graylog itself is using port 9350/tcp for communicating with Elasticsearch (which is using 9300/tcp by default).

https://github.com/Graylog2/graylog2-server/blob/2.2.1/misc/graylog.conf#L291-L292